### DISCUSSION TITLE
useInterval breaking changes between 7.10 and 7.12.2

### DISCUSSION THREAD

**Discussion by grahammatt** [mantinedev/mantine#6819] [Feedback]
I've been trying to figure out why on my own but I have a custom timer component that functioned using the useInterval hook. I cant upgrade to 17.12.2 because the timer is behaving oddly, There are multiple instances of this component and it seems like either only one interval is able to start or no intervals start. 

I have tried rewriting the component to account for the new autoInvoke option but it acts the same. Is there a limit of one interval at a time now?

**Comment by Kenzo-Wada** [comment]
Can you please share a url of reproduction sandbox?
